### PR TITLE
init example for spectra intermezzo

### PR DESCRIPTION
**Pull request by DaanVanHauwermeiren**
First version of a suggested intermezzo for spectral processing.
In the script, there is usage of the following concepts

- custom data types
- specialising functions for these data types
- plot recipe
- fancy data pipelines
- usage of DrWatson to simulate and store simulation data

Not yet in the form of a Pluto notebook, waiting for some content feedback.
I see this as an advanced example for the second day

### CODE REVIEW

File: notebooks/day2/Project.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[deps]
```
Comment (DaanVanHauwermeiren) [inline notebooks/day2/Project.toml:1]: file should be merged with other `Project.toml` files

**Comment by Beramos** [comment]
Wow, that is very long for an intermezzo. Can you see this fit in the 30 min slot allocated for this? https://docs.google.com/spreadsheets/d/1ilkRrCHDSRzEm7S8CZ_NMoiL8Ed5_3MVJaFCLAFgu1c/edit#gid=2059960846

**Comment by Beramos** [comment]
Maybe we can merge it with the macro's intro? #49

**Comment by DaanVanHauwermeiren** [comment]
It has a bit of everything. That's the issue atm.
It uses some macros, but I think it is a little light at this moment to call this an intro to macros

**Comment by DaanVanHauwermeiren** [comment]
Note: force push because of bad credentials

**Comment by Beramos** [comment]
@DaanVanHauwermeiren  I'm going to wait to review this as mentioned in slack we already have way too much content.

**Pull request merged**